projects/fsastorefrontstyles/scss/components/cms/_enriched-responsive-banner.scss

```diff
@@ -2,9 +2,68 @@
   .enriched-banner {
     position: relative;
     width: 100%;
+    &-text {
+      background-color: rgba(255, 255, 255, 0.7);
+      position: absolute;
+    }
+    &-title {
+      font-size: var(--cx-h1-font-size);
+      color: var(--cx-color-secondary);
+      font-weight: var(--cx-font-weight-semi);
+      @include media-breakpoint-down(md) {
+        font-size: 4vw;
+      }
+    }
+    &-details {
+      display: block;
+      margin-top: 1vw;
+      pointer-events: none;
+      font-size: var(--cx-h5-font-size);
+      @include media-breakpoint-down(sm) {
+        font-size: 3.5vw;
+      }
+    }
   }
   /* needed as a fix for Enriched banner not displaying correctly SOMETIMES, when resizing window */
   cx-media img {
     width: 100%;
   }
+  .box-size {
+    max-width: 410px;
+  }
+  .strip-size {
+    width: 100%;
+  }
+  .absolute-center {
+    @include absolute-center;
+  }
+  .vertical-center {
+    @include vertical-center;
+  }
+  .vertical-top {
+    top: 0;
+  }
+  .vertical-bottom {
+    bottom: 0;
+  }
+  .horizontal-right {
```

Comment:
Just `.right`. Also it should only have `right: 0;` declaration. Nothing else. In this case, as we agreed, positioning should be done with margin. We can rethink this approach in some improvement task, but for now let's position it with margin.